Heine.Mvc.ActionFilters.Tests/Services/ObfuscatorServiceTests.cs

```diff
@@ -0,0 +1,109 @@
+﻿using System.Linq;
+using System.Reflection;
+using FluentAssertions;
+using Heine.Mvc.ActionFilters.Services;
+using Heine.Mvc.ActionFilters.Tests.Services.Models;
+using NUnit.Framework;
+
+namespace Heine.Mvc.ActionFilters.Tests.Services
+{
+    [TestFixture]
+    public class ObfuscatorServiceTests
+    {
+        private Assembly documentAssembly;
+
+        [SetUp]
+        public void Init()
+        {
+            // Arrange
+            documentAssembly = typeof(Document).Assembly;
+        }
+
+        [Test]
+        public void ObfuscatorService_ExpandDepthIsFive_ShouldReturnMultiplePaths()
+        {
+            // Act
+            var obfuscationService = new ObfuscaterService(5, documentAssembly);
+
+            var obfuscationGraph = obfuscationService.TypeObfuscationGraphs;
+            var documentGraph = obfuscationGraph[typeof(Document)];
+            var personGraph = obfuscationGraph[typeof(Person)];
+
+            // Assert
+            obfuscationGraph.Should().NotBeEmpty();
+            documentGraph.Should().NotBeEmpty();
+            personGraph.Should().NotBeEmpty();
+            documentGraph.Count.Should().Be(27);
+            documentGraph.FirstOrDefault(s => s.Equals("Content")).Should().NotBeNullOrEmpty();
```

Comment:
Consider to change the assertions to the following format for better readability:
```suggestion
            documentGraph.Should.Contain(s => s.Equals("Content"));
```

Reply:
Good tip. Change has been implemented.